tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -748,7 +748,7 @@ TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_00)
   size_t data_size;
 
   gchar *test_model;
-  guint *sink_called_cnt  = (guint *) g_malloc (sizeof (guint));
+  guint *sink_called_cnt = (guint *) g_malloc0 (sizeof (guint));
```

Comment:
The type casting might be unnecessary here.

> A pointer to void may be converted to or from a pointer to any object type. A pointer to any object type may be converted to a pointer to void and back again; the result shall compare equal to the original pointer.

Reply:
But, this makes build failure:
```
error: invalid conversion from 'gpointer' {aka 'void*'} to 'guint*' {aka 'unsigned int*'} [-fpermissive]
```